src/main/webapp/App.js

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+
+/**
+ * Parent Component that is Rendered to the Page.
```

Comment:
nit: [per javadoc conventions](https://google.github.io/styleguide/javaguide.html#s7.1.2-javadoc-paragraphs) (I realize this is js-doc, but close enough), add an empty line (i.e. one that only contains `*`) between the summary section and the `@param` section.

Reply:
Updated with commit 793c906